pkg/host/machine_info_linux.go

```diff
@@ -13,6 +13,9 @@ import (
 	"regexp"
 	"strconv"
 	"strings"
+
+	"github.com/gobwas/glob"
```

Comment:
This needs to be vendored.
But I also see this is used just to remove some files, so regexp looks like a much better fit.
But since globs can only come from the magical "exclude.txt", I would prefer to find a better to way to do it overall (without the magical exclude.txt file).

Reply:
removed for now